dev/publish_bom.py

```diff
@@ -122,13 +121,14 @@ def publish_changelog_gist(self):
       # Also add some identifying information.
       raw_content_lines[0] = spinnaker_version
       timestamp = '{:%Y-%m-%d %H:%M:%S}'.format(datetime.datetime.now())
-      signature = '\n\nGenerated by {0} at {1}'.format(socket.gethostname(), timestamp)
+      signature = '\n\nGenerated by at {0}'.format(timestamp)
       raw_content_lines.append(signature)
       content = InputFileContent(''.join(raw_content_lines))
       filename = os.path.basename(self.__changelog_file)
       gist = self.__github.get_user().create_gist(True, {filename: content}, description=description)
-      print ('Wrote changelog to Gist at https://gist.github.com/{user}/{id}'
-             .format(user=self.__gist_user, id=gist.id))
+      gist_uri = 'https://gist.github.com/{user}/{id}'.format(user=self.__gist_user, id=gist.id)
+      print ('Wrote changelog to Gist at {0}.'.format(gist_uri))
+      return gist_uri
```

Comment:
You didnt actually write it, you are just initializing the uri.
The write happens later -- and could be written elsewhere.